### PR TITLE
fix skytemp

### PR DESCRIPTION
**Pull request by gabriel-happle**
- fix equation
- use opaque skycover `opaqskycvr_tenths` from weather file directly for calculation
- add documentation

### CODE REVIEW

**Comment by gabriel-happle** [comment]
I'm running the tests to see how much this influences the results.

**Comment by gabriel-happle** [comment]
Since the equation should be correct now, I think we have to accept the changes.
I'm therefore updating the tests.

**Comment by gabriel-happle** [comment]
I don't know why the tests fail. They are passing on my machine.

**Comment by daren-thomas** [comment]
@gabriel-happle it's something with the Jenkins server, I'll check it tomorrow - I can't fix it remotely with TeamViewer for very boring reasons (I deleted the software on my local computer and now can't access it anymore...)

File: cea/utilities/epwreader.py [hunk 1]
```diff
@@ -33,15 +33,26 @@ def epw_reader(weather_path):
...
+    or:
+    https://bigladdersoftware.com/epx/docs/8-6/engineering-reference/climate-calculations.html
+
+    :param Tdrybulb: Dry bulb temperature [C]
+    :param Tdewpoint: Wet bulb temperature [C]
+    :param N: opaque skycover in [tenths], minimum is 0, maximum is 10 see: http://glossary.ametsoc.org/wiki/Sky_cover
+    :return: sky temperature [C]
+    """
+
+    sky_e = (0.787 + 0.764 * math.log((Tdewpoint + 273) / 273)) * (1 + 0.0224 * N - 0.0035 * N ** 2 + 0.00028 * N ** 3)
```
Comment (daren-thomas) [inline cea/utilities/epwreader.py]: I noticed you're dividing by an integer. Let's get in the habit of using floats when we mean floats. I also checked and this file is missing the `from __future__ import division`.

Comment (gabriel-happle) [reply cea/utilities/epwreader.py]: OK. Thanks.

**Pull request merged**